package.json

```diff
@@ -1,21 +1,28 @@
 {
-  "name": "todo",
+  "name": "todo-react",
```

Comment:
I think you mistakingly added two different name, It is better to remove one and stick with just one.